verification/utils/seqs/seqs.gobra

```diff
@@ -36,4 +36,11 @@ requires size >= 0
 ensures len(res) == size
 ensures forall i int :: { res[i] } 0 <= i && i < size ==> res[i] == nil
 decreases _
-pure func NewSeqByteSlice(size int) (res seq[[]byte])
\ No newline at end of file
+pure func NewSeqByteSlice(size int) (res seq[[]byte])
+
+ghost
+requires forall i int :: { &ub[i] } 0 <= i && i < len(ub) ==> acc(&ub[i], _)
+ensures len(res) == len(ub)
+ensures forall i int :: { res[i] } 0 <= i && i < len(ub) ==> res[i] == ub[i]
+decreases _
+pure func ToSeqByte(ub []byte) (res seq[byte])
```

Comment:
if we only use this with `Slices`, why do we have the qps in the precondition?

Reply:
Fixed.